airflow/www/static/js/dag/details/taskInstance/taskActions/ClearInstance.tsx

```diff
@@ -77,6 +78,10 @@ const ClearInstance = ({
   const [failed, setFailed] = useState(false);
   const onToggleFailed = () => setFailed(!failed);
 
+  const initialClearButtonFocusRef = useRef<HTMLButtonElement>(null);
+
+  useKeysPress("shiftKey", ['c', 'C'], onOpen);
```

Comment:
Cool! Do you have a list of what shortcuts you want to do? We'll need to check this across browsers or toggle on/off if it conflicts with a user's shortcuts for something else.

Reply:
Hi @bbovenzi,

Added below shortcuts to the DAGS grid screen - 
- At DagRun
  - Clear existing tasks (`shift + c`)
  - Mark as success or fail (`shift + s` or `shift + f`)
  - Add note (`shift + n`)
- At TaskInstance
  - Clear task (`shift + c`)
  - Mark as success or fail `(shift + s` or `shift + f`)
  - Add note (`shift + n`)

The shortcuts are the same, just depends on what is selected. Will add more in the process.

Yes, I have checked with major browsers (Chrome, Firefox, Edge, Opera), and these shortcuts aren't used as of yet (At least nothing happened when I clicked :)).

Reply:
Hi @bbovenzi,
I have fixed all the issues. Can you have a look once at the implementation?

Reply:
Looking good! Do we want to add a cheatsheet somewhere?
Also, for adding a note, we should make sure to toggle the accordion open if it is closed.

Reply:
Thanks for having a look. 

Yes, working on creating the cheat sheet.

Yes, will make sure to toggle the notes accordion open.

Reply:
> Looking good! Do we want to add a cheatsheet somewhere? Also, for adding a note, we should make sure to toggle the accordion open if it is closed.

Hi @bbovenzi 
- Added a cheat sheet for shortcuts
- Yes, toggling the accordion and making the text area in focus